<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCameraCalibration</class>
 <widget class="QDialog" name="DialogCameraCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>701</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Camera 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Camera 2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenVideoCal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Video </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_VideoCal">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Video CAlibration</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_VideoFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FrameIdx">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>580</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>161</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Number Frames Selected</string>
   </property>
   <widget class="QLabel" name="label_CounterSelectedImage">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>25</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_runCalibration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Calibration</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_selectFrame">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>580</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Frame</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCameraCalibration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCameraCalibration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
